Packs/AzureStorage/ReleaseNotes/1_0_3.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Storage Management
+- Renamed the pack from `Azure Storage` to `Azure Storage Management`.
```

Comment:
```suggestion
- Renamed the *Azure Storage* pack to *Azure Storage Management*.
```

Reply:
Done.
Are there any other changes that have to be made in order to rename the pack except the field in the pack_metadata.json file?